### PR TITLE
Dashboard: code clean-up

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?
This PR makes typescript files from `src/components/*` compliant to typescript and project's tslint rules.

Signed-off-by: Oleksii Kurinnyi <okurinny@redhat.com>

### CODE REVIEW

**Comment by akurinnoy** [comment]
ci-test

**Review by benoitf — APPROVED** [review]

File: dashboard/src/components/api/che-git.spec.ts [hunk 1]
```diff
@@ -9,6 +9,9 @@
  *   Red Hat, Inc. - initial API and implementation
  */
 'use strict';
+import {CheWorkspace} from "./workspace/che-workspace.factory";
```
Comment (ashumilova) [inline dashboard/src/components/api/che-git.spec.ts]: " - > '

**Review by ashumilova — APPROVED** [review]

**Comment by ashumilova** [comment]
Thank you a lot for the clean-up

File: dashboard/src/components/api/che-git.spec.ts (hunk 1 above)
Comment (akurinnoy) [reply dashboard/src/components/api/che-git.spec.ts]: fixed

**Comment by akurinnoy** [comment]
you're welcome :)

**Review by olexii4 — APPROVED** [review]

**Pull request merged**